sysbuild/CMakeLists.txt

```diff
@@ -71,22 +179,42 @@ endfunction(${SYSBUILD_CURRENT_MODULE_NAME}_post_cmake)
 # Consider if this shoulc come through Sysbuild Kconfig flag.
 set(NCS_SYSBUILD_PARTITION_MANAGER TRUE)
 
+list(APPEND CMAKE_MODULE_PATH ${ZEPHYR_NRF_MODULE_DIR}/cmake/sysbuild/modules)
+include(ncs_sysbuild_extensions)
```

Comment:
There's already `nrf/sysbuild/extensions.cmake`. Should those be combined?

Reply:
@tejlmand